### PR TITLE
Prompt users to install the Dart Debug Extension

### PR DESCRIPTION
**Pull request by grouma**
If the extension backend is enabled and the user fails to debug through a local connection, prompt them to install the Dart Debug Extension.

### CODE REVIEW

File: dwds/web/client.dart [hunk 1]
```diff
@@ -187,3 +192,6 @@ void runMain() {
 }
 
 bool get _isChromium => window.navigator.userAgent.contains('Chrome');
+
+const _extensionUri =
+    'https://chrome.google.com/webstore/detail/dart-debug-extension/eljbmlghnomdjgdjmbdekegdkbabckhm';
```
Comment (kevmoo) [inline dwds/web/client.dart:197]: Want me to create a short URL for this?

Let's us get basic analytics and change it after ship!

Comment (kevmoo) [reply dwds/web/client.dart:197]: Use https://goo.gle/2yhG86P

Comment (kevmoo) [reply dwds/web/client.dart:197]: Better yet: https://goo.gle/dart-debug-extension

File: dwds/lib/src/handlers/dev_handler.dart [hunk 2]
```diff
@@ -316,13 +317,23 @@ class DevHandler {
...
+        ..success = false
+        ..promptExtension = false
+        ..error = error);
+      if (_extensionBackend != null) {
+        response = response.rebuild((b) => b
+          ..promptExtension = true
+          ..error = '$error\n\n'
+              'Your workflow alternatively supports debugging through the '
+              'Dart Debug Extension.\n\n'
+              'Would you like to install the extension?');
```
Comment (jakemac53) [inline dwds/lib/src/handlers/dev_handler.dart:334]: link to it?

Comment (jakemac53) [reply dwds/lib/src/handlers/dev_handler.dart:334]: oh nvm I see that this is handled via the prompt

**Comment by grouma** [comment]
Merging on red due to chromedriver mismatch.

**Pull request merged**